### ISSUE TITLE
Matrix Vector multiplication?

### ISSUE THREAD

**Issue by kyunghoon** [mateogianolio/vectorious#152]
i don't seem to see an optimized api call for this, does it exist?

**Comment by mateogianolio** [comment]
That is correct, good feature request. You mean optimized with GEMV? You could also do Matrix Vector multiplication with GEMM (Matrix.multiply)

**Comment by mateogianolio** [comment]
See https://github.com/numpy/numpy/blob/main/numpy/core/src/common/cblasfuncs.c#L208-L700
